server/base/src/main/java/org/apache/accumulo/server/metadata/RootTabletGson.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.accumulo.server.metadata;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import com.google.gson.Gson;
+
+public class RootTabletGson {
+
+  private static Gson gson = new Gson();
+
+  public String location;
+  public Map<String, String> files;
+  public long compactionId;
+
```

Comment:
Might be good idea to embed a version identifier into the serialized JSON, in case the schema changes.